### ISSUE TITLE
ModuleNotFoundError: No module named 'datetime.timezone'; 'datetime' is not a package

### ISSUE THREAD

**Issue by valberg** [adamchainz/django-upgrade#172]
### Python Version

3.10

### Django Version

4.1

### Package Version

1.8.0

### Description

After running the new 1.8.0 my project now throws this error:

```
>>> from datetime.timezone import utc
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'datetime.timezone'; 'datetime' is not a package
```

I guess #169 was a bit too eager since `timezone.utc` is just an attribute on `timezone` and thus not importable.

**Comment by adamchainz** [comment]
Thank you for the report, I guess we both missed that 😅

Fixed in #174, released in 1.8.1!

@hramezani you may want to check my PR, perhaps there's another case it doesn't fix.

**Comment by hramezani** [comment]
Thanks @valberg for reporting. Yes, we missed that.

@adamchainz Thanks for taking care. LGTM :+1:
